lib/report.js

```diff
@@ -73,6 +76,9 @@ class Report {
   }
 
   async run () {
+    if (this.monocartArgv) {
+      return this.runMonocart()
```

Comment:
If we were unable to require Monocart, we should print the install instructions here and exit with 1.

Reply:
I have made the following improvements based on your suggestion
- Move the import library to try/catch.
- Use the `import("xxx")` function to load the lib asynchronously. It should be good enough without creating a new help file, and added new test case to cover it (rename the lib entry file can mock it).
- Print the install instructions and exit with 1 when failed to load the lib.

Please review, thanks.